packages/terra-functional-testing/src/reporters/file-output-reporter/index.js

```diff
@@ -0,0 +1,207 @@
+const fs = require('fs-extra');
+const path = require('path');
+const stripAnsi = require('strip-ansi');
+const SpecReporter = require('@wdio/spec-reporter').default;
+const endOfLine = require('os').EOL;
+const { Logger } = require('@cerner/terra-cli');
+
+const LOG_CONTEXT = '[terra-functional-testing:file-output-reporter]';
+
+class FileOutputReporter extends SpecReporter {
+  constructor() {
+    super();
+    this.runners = [];
+    this.resultJsonObject = {
+      startDate: '',
+      type: 'wdio',
+      locale: '',
+      formFactor: '',
+      theme: '',
+      output: {},
+      endDate: '',
+    };
+    this.fileName = '';
+    this.moduleName = process.cwd().split(path.sep).pop();
+
+    this.setResultsDir = this.setResultsDir.bind(this);
+    this.hasResultsDir = this.hasResultsDir.bind(this);
+    this.setTestModule = this.setTestModule.bind(this);
+    this.printReport = this.printReport.bind(this);
+    this.getMessage = this.getMessage.bind(this);
+
+    this.setResultsDir();
+    this.hasResultsDir();
+  }
+
+  /**
+   * Sets results directory for the test run. Uses the wdio reporterOptions.outputDir if set, otherwise
+   * it outputs to tests/wdio/reports.
+   * @return null;
+   */
+  setResultsDir() {
+    this.resultsDir = path.join(process.cwd(), 'tests', 'wdio', 'reports');
```

Comment:
Do we need to handle the reports path for mono-repos or will the report for all packages be in the root level?

Reply:
reports for all the packages are in the root level